torch/csrc/jit/tensorexpr/ir_simplifier.cpp

```diff
@@ -1077,6 +1077,27 @@ Stmt* PolynomialTransformer::mutate(const For* v) {
   return new For(var_new, start_new, stop_new, body_new, loop_options);
 }
 
+Stmt* PolynomialTransformer::mutate(const Block* v) {
```

Comment:
Nit: it started as `PolynomialTransformer`, but now it does all kinds of simplification. Maybe we should rename it at some point?

Reply:
Do you have an idea for a better name? This the first of the two phases of the IR simplifier.

Reply:
Not really, maybe something general like `IRSimplifierImpl`. But that's minor anyway.